### DISCUSSION TITLE
Periodogram has issues with method='lombscargle'

### DISCUSSION THREAD

**Discussion by rfreire-400** [lightkurve/lightkurve#1343] [Q&A] [answered]
I'm trying to create a periodogram but keep getting this issue:
<img width="876" alt="Screenshot 2023-06-06 at 2 59 02 PM" src="https://github.com/lightkurve/lightkurve/assets/113457579/75015933-9959-4958-9d6f-28ff6e6978f3">

It only seems to do with when method = lombscargle and does not do it with BoxLeastSquares
<img width="876" alt="Screenshot 2023-06-06 at 2 57 12 PM" src="https://github.com/lightkurve/lightkurve/assets/113457579/f973df3a-b4ca-4be6-8c24-4910be033a7a">

I'm running this on Python 3.9.6 and Lightkurve 2.4.0. Anyone knows why this is happening?

**Comment by christinahedges** [comment, marked as answer]
Thanks for opening a discussion! This is now issue #1344
